### PR TITLE
Add feature to generate an image CSV file.

### PR DESCRIPTION
**Pull request by cfezequiel**
## Description

Adds feature to create an image CSV file from an image directory.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## Checklist

Please delete options that are not relevant.

- [X] My code adheres to the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
- [X] I ran `make pylint` and code is rated 10/10
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I ran `make test` and all tests pass
- [X] I ran the tool and verified the change works
- [X] I have adequately commented my code, particularly in hard-to-understand areas
- [X] I have made relevant changes to the documentation, if needed
- [X] My changes generate no new warnings
- [X] I have corrected any misspellings in my code
- [X] (For feature) I made sure that I'm merging to the `dev` branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +194,27 @@ tfrecorder inspect \
     --output_dir=/tmp/output
 ```
 
+### Creating an image CSV file
+
```
Comment (mbernico) [inline README.md:198]: Do we want to note how the labels and splits are generated here?

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
-apache-beam[gcp] >= 2.22.0
+avro == 1.10.0
+avro-python3 == 1.9.2.1
+apache-beam[gcp] == 2.24.0
 pandas >= 1.0.4
-tensorflow_transform >= 0.22
+# TODO(cezequiel): Support breaking changes in TF Transform 0.25.0
```
Comment (mbernico) [inline requirements.txt:5]: That's unfortunate... Thanks TFT Team :)

File: tfrecorder/converter.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
 import logging
 import os
-from typing import Any, Dict, Optional, Sequence, Tuple, Union
+from typing import Any, Dict, Optional, Sequence, Union
```
Comment (mbernico) [inline tfrecorder/converter.py:25]: I think internally I've seen us put each of these on a single line.  

eg 
from typing import Any
from typing import Dict
etc...

Do you think we should move towards that format here?

**Review by mbernico — COMMENTED** [review]
One note on documentation, otherwise looks okay.

File: README.md (hunk 1 above)
Comment (cfezequiel) [reply README.md:198]: Makes sense. Maybe we could just reference the image directory section for the schema.

File: tfrecorder/converter.py (hunk 3 above)
Comment (cfezequiel) [reply tfrecorder/converter.py:25]: Normally, we should be importing up to the module level only, but there's an explicit exception for the `typing` module:
https://google.github.io/styleguide/pyguide.html#typing-imports

**Review by mbernico — APPROVED** [review]
lgtm

**Pull request merged**